csi/controllerserver.py

```diff
@@ -301,7 +301,7 @@ def DeleteVolume(self, request, context):
 
         delete_volume(request.volume_id)
         logging.info(logf(
-            "Volume deleted",
+            "Delete Volume response completed",
```

Comment:
@aravindavk Checking for hostvol & then pvReclaimPolicy type would require search_volume() again, so specific logs will be made from volumeutils.py itself

Reply:
ok